src/shared/libraries/Emitter.ts

```diff
@@ -41,10 +41,12 @@ export default class Emitter {
    * execute only once.
    */
   public once(event: string, listener: EventHandler): Emitter {
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
     const that = this;
 
-    function fn(this: Function) {
+    function fn(this: unknown) {
```

Comment:
There are 18 new `unknown` types in this commit. I think we can fix at least some of these such as here

Reply:
This one is fixable, it should be `EventHandler`, fixed it in commit 543f21a59cf5b472a9f9535cc2f3cbeef23beea4.

Most of the other ones are an improvement though still, example `Function` was turninged into `(param: unknown) => void` which is more accurate.
* `Function` is a banned type with the [default eslint rules](https://typescript-eslint.io/rules/ban-types/)

A few of the `unknowns` could probably be replaced with generics, however that will require adding new types which seems out of scope for this PR.